constructor-args/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+#![no_std]
+use soroban_sdk::{contract, contractimpl, symbol_short, Env, Symbol};
+
+#[contract]
+pub struct ImmutableCounter;
+const COUNTER: Symbol = symbol_short!("COUNTER");
+
+#[contractimpl]
+impl ImmutableCounter {
```

Comment:
The way we did this is pretty funny but it'd be so much better if it was either renamed or redone to actually allow incrementing so that the name itself isn't an oxymoron

Reply:
good idea; just updated the main increment contract instead